configuration/ibm/50001001.json

```diff
@@ -876,24 +876,29 @@
         "/sys/bus/i2c/drivers/at24/7-0051/eeprom": [
             {
                 "inventoryPath": "/system/chassis/motherboard/lcd_op_panel_hill",
-                "devAddress": "7-0051",
-                "driverType": "at24",
-                "busType": "i2c",
-                "presence": {
-                    "pollingRequired": true,
-                    "pin": "RUSSEL_OPPANEL_PRESENCE_N",
-                    "value": 0,
-                    "gpioI2CAddress": "0-0020"
-                },
+                "gpioI2CAddress": "0-0020",
```

Comment:
why is this not under presence or set gpio ?

Reply:
This information is not required for that operation. This has been added just because we need this information in error log. Hence kept generic.

Reply:
As decided, will get rid of this.